### PR TITLE
Dev/manipulation polish

### PR DESCRIPTION
**Pull request by jamesashley1**
## Overview

High level overview of this pull request.
Changes to ManipulationHandler to allow scaling to be continuous even if pressure (input) doesn't change. Create a virtual method, CalculateScaleAmount, so classes that inherit from this can override the scaling logic.

### CODE REVIEW

File: XRTK.SDK/Packages/com.xrtk.sdk/Features/Input/Handlers/ManipulationHandler.cs [hunk 1]
```diff
@@ -627,6 +628,16 @@ protected virtual float CalculateNudgeDistance(InputEventData<Vector2> eventData
             return prevExtent + nudgeAmount * (eventData.InputData.y < 0f ? -1 : 1);
         }
 
+        protected virtual Vector3 CalculateScaleAmount(InputEventData<Vector2> eventData, Vector3 prevScale)
```
Comment (StephenHodgson) [inline XRTK.SDK/Packages/com.xrtk.sdk/Features/Input/Handlers/ManipulationHandler.cs]: Wondering if we should do the same for the rotation calculation as well, thoughts?

Comment (SimonDarksideJ) [reply XRTK.SDK/Packages/com.xrtk.sdk/Features/Input/Handlers/ManipulationHandler.cs]: That would be a good idea, yes

File: XRTK.SDK/Packages/com.xrtk.sdk/Features/Input/Handlers/ManipulationHandler.cs [hunk 2]
```diff
@@ -368,6 +374,16 @@ protected virtual void Update()
             {
                 primaryPointer.PointerExtent = updatedExtent;
             }
+
+            if (IsPressed && IsScalingPossible)
```
Comment (StephenHodgson) [inline XRTK.SDK/Packages/com.xrtk.sdk/Features/Input/Handlers/ManipulationHandler.cs]: This should likely be above in the `if (IsBeingHeld)` check

File: XRTK.SDK/Packages/com.xrtk.sdk/Features/Input/Handlers/ManipulationHandler.cs [hunk 3]
```diff
@@ -368,6 +374,16 @@ protected virtual void Update()
...
+            {
+                manipulationTarget.position = grabbedPosition + updatedPointerPosition;
```
Comment (StephenHodgson) [inline XRTK.SDK/Packages/com.xrtk.sdk/Features/Input/Handlers/ManipulationHandler.cs]: I'd recommend just using the primaryPointer's current pointer position.

File: XRTK.SDK/Packages/com.xrtk.sdk/Features/Input/Handlers/ManipulationHandler.cs (hunk 1 above)
Comment (StephenHodgson) [reply XRTK.SDK/Packages/com.xrtk.sdk/Features/Input/Handlers/ManipulationHandler.cs]: We can add this later if needed.

**Pull request merged**